tests/clickhouse-test

```diff
@@ -920,6 +920,9 @@ class SettingsRandomizer:
         "optimize_functions_to_subcolumns": lambda: random.randint(0, 1),
         "parallel_replicas_local_plan": lambda: random.randint(0, 1),
         "output_format_native_write_json_as_string": lambda: random.randint(0, 1),
+        "parallel_replicas_min_number_of_rows_per_replica": lambda: random.randint(
+            0, 1
```

Comment:
Why not to give a bigger range to it? Like `[0,5]`?

Reply:
The goal is only to enable "rows to read" analysis w/o disabling parallel replicas. And with `parallel_replicas_min_number_of_rows_per_replica=1`, the PR should be disabled only if a query going to read only 1 row.